src/screens/OrganizationActionItems/ActionItemViewModal/ActionItemViewModal.tsx

```
@@ -154,36 +151,21 @@ const ItemViewModal: FC<IViewModalProps> = ({ isOpen, hide, item }) => {
-              style: {
-                WebkitTextFillColor: isCompleted
-                  ? 'green'
-                  : 'var(--pendingStatus-color)',
-              },
-            }}
-            disabled
-          />
+        <Form.Group className="d-flex gap-3 mx-auto mb-3 align-items-start w-100">
+          {/* Status using the shared StatusBadge component (replaces ad-hoc icon + color styles) */}
+          <FormControl fullWidth>
+            <label className="form-label mb-2">{t('status')}</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)